testsuite/integration/clustering/pom.xml

```diff
@@ -45,7 +45,7 @@
         <surefire.forked.process.timeout>5400</surefire.forked.process.timeout>
         <test-group>org/jboss/as/test/clustering/cluster</test-group>
         <version.org.jboss.byteman>4.0.6</version.org.jboss.byteman>
-        <version.org.infinispan.server>11.0.4.Final</version.org.infinispan.server>
+        <version.org.infinispan.server>11.0.3.Final</version.org.infinispan.server>
```

Comment:
We really do need to use 11.0.4.Final - because 11.0.3.Final suffers from the shutdown problem fixed by https://issues.redhat.com/browse/ISPN-12374.